docs/source/quickstart.rst

```diff
@@ -0,0 +1,194 @@
+
+    # Return the result in a numpy.array (fits in memory)
+    >>> np_da_y = np.array(da_y)
+    # or
+    >>> da_y.compute()
+
+If it doesn't, you can store the result to a file:
+
+.. code-block:: python
+
+    # Store the result to disk
+    >>> import h5py
+
+    >>> da_y.to_hdf5('myfile.hdf5', 'result')
+
+You can also create arrays from files:
+
+.. code-block:: python
+
+    # Read results from file
+    >>> f = h5py.File('myfile.hdf5')
+    >>> dset = f['/result']
+    >>> da_result = da.from_array(dset, chunks=(1000, 1000))
+
+
+For more information on ``dask.array`` visit the array_ section. The reference guide in the API_ section provides a detailed description of all the available methods and their arguments.
+
+.. _FAQ: faq.rst
+.. _array: array-overview.rst
+.. _api: array-api.rst
+
+
+**dask.dataframe:**
+
+Equivalently, a ``dask.dataframe`` is similar to a ``pandas.Dataframe`` with some slight alterations due to the parallel nature of dask.
+
+.. code-block:: python
+
+    >>> import pandas as pd
+    >>> import dask.dataframe as dd
+    
+    >>> df = pd.read_csv('iris.csv')
+    >>> da_df = dd.read_csv('iris.csv')
+
+You can then interact with the dataframe in a similar manner.
+
+.. code-block:: python
+
+    # In pandas
+    >>> df.head()
+
+    # In dask
+    >>> ddf.head()
+
+With all dask collections (e.g. Array, Bag, DataFrame) one triggers computation by calling the ``.compute()`` method.
+
+.. code-block:: python
+
+    # In pandas
+    >>> max_sepal_length_setosa = df[df.species == 'setosa'].sepal_length.max()
+    >>> max_sepal_length_setosa
+    5.7999999999999998
+
+    # In dask
+    >>> d_max_sepal_length_setosa = ddf[ddf.species == 'setosa'].sepal_length.max()
+    >>> d_max_sepal_length_setosa.compute()
+    5.7999999999999998
+
+
+The ``pandas.DataFrame`` API is huge and dask currently doesn't support it entirely, but implements a subset in ``dask.dataframe``. For a list of the current supported API, take a look at the *What definetely works* section_. The reference guide for the dask.dataframe API provides a detailed description of all the available methods and their arguments.
+
+.. _section: dataframe.rst
+.. _API: dataframe-api.rst
+
+
+**dask.bag:**
+
+If you are working with semi-structure data, like JSON blobs or log files, ``dask.bag`` will provide a nice interface to perform parallel computations on it.
+
+You can create a bag from a Python sequence (``db.from_sequence``), from lines in many files (``db.from_filenames``) and from HDFS files (``db.from_hdfs``).
+
+.. code-block:: python
+
+    >>> import dask.bag as db
+    >>> import json
+
+    # Get tweets as a dask.bag from compressed json files
+    >>> b = db.from_filenames('*.json.gz').map(json.loads)
+
+    # Take two items in dask.bag
+    >>> b.take(2)
+
+    # Count the frequencies of user locations
+    >>> freq = b.pluck('user').pluck('location').frequencies()
+
+    # Transform to a dask.dataframe
+    >>> df = freq.to_dataframe()
```

Comment:
I wonder if we can combine this with a dataset on s3 so that we can have something actually runable